pkgs/by-name/so/sobjectizer/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  withStatic ? true,
+  withShared ? true,
```

Comment:
```nix
  withStatic ? stdenv.hostPlatform.isStatic,
  withShared ? !withStatic,
```

`stdenv.hostPlatform.isStatic` is false by default, so `withShared` becomes true, which leads to the tests being run. 😉

Reply:
Also consider adding `assert buildExamples -> withStatic;`

Reply:
> ```nix
>   withStatic ? stdenv.hostPlatform.isStatic,
>   withShared ? !withStatic,
> ```
> 
> `stdenv.hostPlatform.isStatic` is false by default, so `withShared` becomes true, which leads to the tests being run. 😉

Oh, sorry for unnecessary noise. This changes were lost during "fixing history" process :) Fixed.